os-eap7-launch/added/launch/messaging.sh

```diff
@@ -313,14 +315,19 @@ function inject_brokers() {
       # topics environment variable name format: [NAME]_[BROKER_TYPE]_TOPICS
       topics=$(find_env "${prefix}_TOPICS")
 
+      tracking=$(find_env "${prefix}_TRACKING")
+      if [ -n "${tracking}" ]; then
+         ra_tracking="tracking=\"${tracking}\""
+      fi
```

Comment:
ra_tracking is pontentially undefined here, it would be better to explicitly set it to an empty string, just to be safe from a global "ra_tracking" set somewhere else...

Reply:
It is a hard scenario but can happen since this var is not local, I'll updated it, does it block the QE to test using the images that is already there?

Reply:
no I don't think we need to retest on trivial changes.